### PR TITLE
Non root node exporter

### PR DESCRIPTION
**Pull request by corest**
Towards https://github.com/giantswarm/giantswarm/issues/5983

### CODE REVIEW

File: helm/kubernetes-node-exporter-chart/charts/migration/templates/rbac.yaml [hunk 1]
```diff
@@ -33,6 +33,10 @@ rules:
   verbs:
   - "get"
   - "delete"
+- apiGroups:     ['extensions']
```
Comment (pipo02mix) [inline helm/kubernetes-node-exporter-chart/charts/migration/templates/rbac.yaml]:
```suggestion
- apiGroups:     
  - extensions
```

File: helm/kubernetes-node-exporter-chart/charts/migration/templates/rbac.yaml [hunk 2]
```diff
@@ -33,6 +33,10 @@ rules:
...
+  resources:     ['podsecuritypolicies']
```
Comment (pipo02mix) [inline helm/kubernetes-node-exporter-chart/charts/migration/templates/rbac.yaml]:
```suggestion
  resources:     
  -podsecuritypolicies
```

File: helm/kubernetes-node-exporter-chart/charts/migration/templates/rbac.yaml [hunk 3]
```diff
@@ -33,6 +33,10 @@ rules:
...
+  resourceNames: ['{{ .Values.name }}']
```
Comment (pipo02mix) [inline helm/kubernetes-node-exporter-chart/charts/migration/templates/rbac.yaml]:
```suggestion
  resourceNames:     
  - '{{ .Values.name }}'
```

File: helm/kubernetes-node-exporter-chart/charts/migration/templates/rbac.yaml [hunk 4]
```diff
@@ -33,6 +33,10 @@ rules:
...
+  verbs:         ['use']
```
Comment (pipo02mix) [inline helm/kubernetes-node-exporter-chart/charts/migration/templates/rbac.yaml]:
```suggestion
  verbs:     
  - use
```

**Review by pipo02mix — APPROVED** [review]
LGTM just some formatting

**Review by calvix — APPROVED** [review]

**Pull request merged**